src/backend/distributed/sql/udfs/citus_stat_activity/latest.sql

```diff
@@ -0,0 +1,24 @@
+--DROP VIEW IF EXISTS pg_catalog.citus_stat_activity;
+--DROP TYPE IF EXISTS pg_catalog.citus_stat_activity_one_node;
+
+CREATE TYPE citus.citus_stat_activity_one_node AS (global_pid bigint, worker_query boolean, pg_stat_activity pg_stat_activity);
+
+ALTER TYPE citus.citus_stat_activity_one_node SET SCHEMA pg_catalog;
+GRANT ALL ON TYPE pg_catalog.citus_stat_activity_one_node TO PUBLIC;
+
+CREATE VIEW citus.citus_stat_activity AS
+SELECT all_csa_1n.global_pid, nodeid, all_csa_1n.worker_query, (all_csa_1n.pg_stat_activity).*
```

Comment:
would it make sense to call this `is_worker_query` to make it clear it's not a query?